scheds/rust/scx_layered/src/bpf/intf.h

```diff
@@ -88,19 +90,21 @@ struct cpu_ctx {
 	u64			lstats[MAX_LAYERS][NR_LSTATS];
 	u64			ran_current_for;
 	u32			layer_idx;
-	u32			node_idx;
 	u32			cache_idx;
+	u32			node_idx;
```

Comment:
Did these swap for a reason?

Reply:
was thinking going from the least specific to the most specific for ordering, but either way is fine